modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/js/ckeditor5/ClassicEditor.tsx

```diff
@@ -0,0 +1,69 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {CKEditor} from '@ckeditor/ckeditor5-react';
+import {
+	Bold,
+	ClassicEditor as BaseClassicEditor,
```

Comment:
The default editor has the `powered by CKEditor` logo on bottom right: 
![image](https://github.com/user-attachments/assets/c6efd68f-0b59-4ed2-9ea5-4ea3998dadb1)


The legitimate way to remove it to [get commercial key](https://ckeditor.com/docs/ckeditor5/latest/getting-started/setup/managing-ckeditor-logo.html#how-to-remove-the-powered-by-ckeditor-logo). Do we have one?

Reply:
I am checking